misskey-websocket/src/client/builder.rs

```diff
@@ -20,12 +26,22 @@ pub struct WebSocketClientBuilder {
 }
 
 trait ResultExt<T, E> {
+    fn err_into<U>(self) -> StdResult<T, U>
```

Comment:
Can you replace the uses of `map_err(Into::into)` in this module (i.e. l68) with `err_into`?